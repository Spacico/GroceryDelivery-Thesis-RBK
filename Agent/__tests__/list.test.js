import React from 'react';
import renderer from 'react-test-renderer';
import List from '../Components/list';
import Mapo from '../Components/mapView';
import getDirections from 'react-native-google-maps-directions';

describe('list component Testing', () => {
    it('renders with out crashing', () => {
        const rendered = renderer.create(< List/>).toJSON();
        expect(rendered).toBeTruthy();
    });
    it('contains TextInput', () => {
        const TextInput = require('TextInput');
        const Tree = renderer.create(<TextInput />).toJSON();
        expect(Tree).toMatchSnapshot();
    });
    it('contains TouchableOpacity', () => {
        const TouchableOpacity = require('TouchableOpacity');
        const Tree = renderer.create(<TouchableOpacity />).toJSON();
        expect(Tree).toMatchSnapshot();
    });
    it('contains View', () => {
        const View = require('View');
        const Tree = renderer.create(<View />).toJSON();
        expect(Tree).toMatchSnapshot();
    });
    // it('contains ScrollView', () => {
    //     const ScrollView = require('Modal');
    //     const Tree = renderer.create(<Modal />).toJSON();
    //     expect(Tree).toMatchSnapshot();
    // });
    it('contains Mapo',()=>{
        const Mapo = renderer.create('Mapo');
        const Tree = renderer.create(<Mapo/>).toJSON();
        expect(Tree).toMatchSnapshot();
    })
})