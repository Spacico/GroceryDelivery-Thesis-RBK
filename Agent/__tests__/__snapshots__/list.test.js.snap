// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list component Testing contains ScrollView 1`] = `
<RCTScrollView>
  <View />
</RCTScrollView>
`;

exports[`list component Testing contains TextInput 1`] = `<TextInput />`;

exports[`list component Testing contains TouchableOpacity 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
/>
`;

exports[`list component Testing contains View 1`] = `<View />`;
